### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#3129] [Announcements]
## Windows App SDK 1.2 (1.2.221109.1)

WinAppSDK 1.2.0 is the latest stable version of WinAppSDK providing new features and improvements to the platform.

**New features include:**
- 3P developers can create widgets for Windows 11 devices
- WinUI 3 apps can play audio and video with media playback controls (MediaPlayerElement + MediaTransportControl)
- App support for Dynamic Refresh Rate (DRR) to automatically switch between 60hz and 120hz refresh rates
- WinUI 3 controls, styles, and behavior are up to date with WinUI 2.8 (InfoBadge)
- Simpler notification creation with AppNotificationBuilder
- HDR and Auto Color Management for desktop apps are now supported via DisplayInformation APIs

**Additional updates include:**
- 11% runtime size reduction in 1.2.0 from Windows App SDK 1.1.5 on x64
- Support for .NET 7
- Support for building native apps on Arm64 Visual Studio 17.4
- AppWindow title bar APIs available downlevel to Windows 10 1809.
- App size reduction for C# apps published with IL trimming

**To see everything that's new and changed, see the full [Windows App SDK 1.2 release notes](https://aka.ms/windowsappsdk/1.2/1.2.0/release-notes).**  

### Try it out
- Download the [1.2.221109.1 NuGet](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.2.221109.1) package to use WinAppSDK 1.2 in your app.
- Build samples from the [WinAppSDK-Samples](https://github.com/microsoft/WindowsAppSDK-Samples) repo to try out components across the platform.
- Download and update the [WinUI Gallery](https://www.microsoft.com/store/productId/9P3JFPWWDZRC) to see the WinUI 3 updates firsthand.

### Getting Started 

To get started using Windows App SDK to develop Windows apps, check out the following documentation:    
- [Install developer tools ](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment).     
- [Create a WinUI 3 app](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app).   
- [Continue your development journey](https://docs.microsoft.com/windows/apps/develop).   

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by harvinders** [comment]
@gabbybilka The link https://aka.ms/windowsappsdk/1.1/1.2.0/release-notes is not working.

**Reply by gabbybilka** [reply]
Thanks 😊 Fixed! https://aka.ms/windowsappsdk/1.2/1.2.0/release-notes

**Reply by triplef** [reply]
@gabbybilka maybe you can fix the link here as well:
https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.2.0

**Reply by gabbybilka** [reply]
Good callout, thanks. I wish there was an option for them to be linked as the discussion is generated from the release. Someday 😊
